spanner/api/Spanner.Samples/Performance.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_insert_data_with_timestamp_column]
+
+public class Performance
```

Comment:
Same comment as with the Album class, I actually think it's better if we add them as inner classes of all the sample classes that need it.